frontend_tests/node_tests/search_suggestion.js

```diff
@@ -526,9 +527,9 @@ run_test('has_suggestions', () => {
         '-has:attachment',
     ];
     assert.deepEqual(suggestions.strings, expected);
-    assert.equal(describe('-has:link'), 'Exclude messages with one or more link');
-    assert.equal(describe('-has:image'), 'Exclude messages with one or more image');
-    assert.equal(describe('-has:attachment'), 'Exclude messages with one or more attachment');
+    assert.equal(describe('-has:link'), 'Translated: exclude translated: messages with one or more link');
```

Comment:
I had to add two `translated` tags here because the string gets translated twice. Not sure if there is a workaround for this.